src/main/java/mate/academy/boot/bootdemo/controllers/ProductController.java

```diff
@@ -30,8 +30,8 @@ public void add(@RequestBody @Valid ProductDto productDto) {
         productService.save(productMapper.getProduct(productDto));
     }
 
-    @GetMapping("/{id}")
-    public ProductDto getProductById(@PathVariable Long id) {
+    @GetMapping("/product")
```

Comment:
```suggestion
    @GetMapping("/{id}")
```
https://restfulapi.net/resource-naming/